subprojects/packagefiles/iir/meson.build

```diff
@@ -0,0 +1,20 @@
+project('iir', 'cpp',
+        version : '1.9.2',
+        license : 'MIT',
```

Comment:
We can also add a `meson_version: '>=0.50.0'` kwarg to the `project()` function. Currently, the two most recent features this wrap uses are:

```
iir| * 0.46.0: {'pkgconfig.generate optional positional argument'}
iir| * 0.50.0: {'CMake Module'}
```

Reply:
(If you remove use of the cmake module, you can set it to 0.46.0 instead of 0.50.0.)

Reply:
Thanks for all the pointers @eli-schwartz ; will get these addressed.

Reply:
Added `meson_version: '>=0.46.0`'